client/src/templates/Challenges/quiz/exit-quiz-modal.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { createSelector } from 'reselect';
+import { useTranslation } from 'react-i18next';
+import { Button, Modal, Spacer } from '@freecodecamp/ui';
+
+import { closeModal } from '../redux/actions';
+import { isExitQuizModalOpenSelector } from '../redux/selectors';
+
+interface ExitQuizModalProps {
+  closeExitQuizModal: () => void;
+  isExitQuizModalOpen: boolean;
+  onExit: () => void;
+}
+
+const mapStateToProps = createSelector(
+  isExitQuizModalOpenSelector,
+  (isExitQuizModalOpen: boolean) => ({
+    isExitQuizModalOpen
+  })
+);
```

Comment:
```suggestion
const mapStateToProps = (state: unknown) => ({
  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
  isExitQuizModalOpen: isExitQuizModalOpenSelector(state)
});
```
We do this all over the place, but `createSelector` creates a memoized selector. If we're not computing anything, we don't need to memoize, it just creates an extra layer of abstraction.